unique_toolkit/unique_toolkit/language_model/service.py

```diff
@@ -8,10 +8,10 @@
 from unique_toolkit.content.schemas import ContentChunk
 from unique_toolkit.language_model.infos import LanguageModelName
 from unique_toolkit.language_model.schemas import (
-    LanguageModelMessages,
     LanguageModelResponse,
     LanguageModelStreamResponse,
     LanguageModelTool,
+    Messages,
```

Comment:
Maybe I miss the point why to switch here. Cant we just update LanguageModelMessages? Otherwise we have to also adjust all modules that already use these function